Upgrading xmonad setup with taffybar

norrs opened the pull request:
File based on Ondřej Súkup @mimi1vx work

Commit:
Upgrading xmonad setup with taffybar

File based on Ondřej Súkup @mimi1vx work

norrs commented on dot.xmonad/xmonad.hs:
```diff
@@ -30,29 +59,189 @@ myKeys conf = M.fromList
+                ,manageHook = myManageHook
+                ,handleEventHook = myEventHook
+                ,startupHook = myStartupHook
+                ,keys = keys defaultConfig <+> myKeys } `additionalKeys`
```
This is probably wrong, doesn't work :(

mimi1vx replied:
remove keys + myKeys

norrs commented on dot.xmonad/xmonad.hs:
```diff
@@ -30,29 +59,189 @@ myKeys conf = M.fromList
+     spawnOnce "xrdb -merge ~/.Xdefaults"
+     spawnOnce "xsetroot -solid black"
+     spawnOnce "status-notifier-watcher"
+     spawnOnce "taffybar"
```
These programs doesn't boot..

mimi1vx replied:
L161 and L162 can be changed to `spawn`

norrs reviewed (commented):
cc @mimi1vx :p

Commit:
Newer xmonad setup creates an xmonad.state and promp-history file, ignore please

Commit:
Still need to run some stuff from .xsession ...

xrdb merge resources, status-notifier-watcher for taffybar which replaces xmobar
xsetroot is imortant for taffybar to behave at all

Commit:
spawnOnce doesnt work for me.. and skip moving terminals to desktop 1

Commit:
Add iconFile for my terminal

Commit:
Updates to my xmonad.hs config

norrs commented:
@norrs: can I ping myself to make a notification on my own?

Commit:
Larger font for prompt config

Commit:
Upgrade to latest xmonad

* Need to use fulLScreenFloat in layouts, and myBorders has renamed the
OnlyFloat to OnlyScreenFloat.

We remove manageDocks and add the docks $ hook

Commit:
Trick newest idea and java to wokr better with tile based wms

Commit:
Add .lib and .config/rofi to install

Commit:
get_cols script from Ivan Malison

Make it available as a function get_cols in bash via
dot.lib/shellenv/functions.sh which is sourced in bash

Commit:
My rofi config for now

Commit:
Kubetail autocomplete

Commit:
Add an sample xmonad file for reference for later

Commit:
source_directory_files function to source /home/norangshol/.lib/shellenv

Commit:
Load the shell libs in .xsession, verify later if needed?

Commit:
Have systemd for user import the PATH loaded from .pathrc

Commit:
Clean up my .xsession file, and we're now using systemctl --user to spawn processes

norrs commented:
Merging this in favor of https://github.com/norrs/dotfiles/pull/1 :P
